packages/svelte-undp-design/src/lib/Header/Header.svelte

```diff
@@ -138,7 +138,7 @@
 		</div>
 
 		{#if showProgressBar}
-			<progress class="progress is-small is-info" max="100" />
+			<progress style="height:0.2rem!important;" class="progress is-small is-info" max="100" />
```

Comment:
This probress bar is from Bulma.

https://bulma.io/documentation/elements/progress/

if you are going to add smaller than `is-small` option, could you add another new class like `is-xsmall`?

probably we can add new parameter for this header to select progress size according to bulma.